landscape/client/monitor/computertags.py

```diff
@@ -15,7 +15,7 @@ class ComputerTags(DataWatcher):
     run_interval = 3600  # Every hour only when data changed
     run_immediately = True
 
-    def __init__(self, args=sys.argv):
```

Comment:
Regarding these three files, can we try to fix the issues instead of bypassing TICS?

Reply:
the `args=sys.argc` could probably be fixed easily; I'm hesitant to change the other two, though.  The compatibility file necessarily has some Python 2 code, so TICS will complain about that when it runs under Python 3.  `os._exit(0)` has somewhat unique behavior over other system exits, so I don't want to modify that.

In general I prefer to keep the scope of this merge narrow so that there are only CI changes.  IMO another MR would be best to clean up the suppressions with code changes.

Reply:
Sure, let's keep it CI specific but follow it up with a ticket.

Reply:
Snuck in a fix for this so we don't have to suppress it